lib/curation_concerns/version.rb

```diff
@@ -1,3 +1,3 @@
 module CurationConcerns
-  VERSION = '1.7.5'.freeze
+  VERSION = '1.7.7'.freeze
```

Comment:
1.7.6 got realeased without the version change getting merged back into the 1-7-stable branch for some reason.  The current version is definitely 1.7.6.  So bumping to 1.7.7 is correct.